pkg/ccl/changefeedccl/sink_cloudstorage.go

```diff
@@ -55,6 +56,17 @@ func cloudStorageFormatTime(ts hlc.Timestamp) string {
 	return fmt.Sprintf(`%s%09d%010d`, t.Format(f), t.Nanosecond(), ts.Logical)
 }
 
+func generateDataFilePartition(format string, timestamp time.Time) string {
+	folderPath := format
+	folderPath = strings.ReplaceAll(folderPath, "%d", timestamp.Format("2006-01-02"))
```

Comment:
I wonder if there would be any benefit to aligning this with strftime?  I believe that would mean this changes to `%F` and the rest can remain the same.

https://man7.org/linux/man-pages/man3/strftime.3.html